deploy/operations/monitoring.md

```diff
@@ -46,3 +46,338 @@ You will need to change the values in the `prometheus` fields in your metadata t
+    2. USS did not configure their system to use features (e.g., ASTM F3548-21 strategic coordination) requiring a DSS in the test of their system in the pool environment
+8. DSS instance is working, but another part of the owning USS's system has failed
+    1. USS deploys their DSS instance differently than/separately from the rest of their system, and the rest-of-system deployment failed while the DSS instance deployment is unaffected
+    2. A component in the rest of the USS's system failed
+9. Database software indicates success to the core-service client, but does not correctly synchronize data to other DSS instances
+    1. There is a critical bug in the database software (this would seem to be a product problem rather than a configuration problem)
+10. Aux API works but SCD/RID API does not work or is disabled
+    1. DSS instance configuration does not enable SCD/RID APIs as needed
+    2. SCD/RID endpoint routing does not work (though other routing does work)
+11. Database nodes are unavailable such that quorum is not met for certain ranges
+    1. Database node container(s) run out of disk space
+    2. Database node container(s) are shut down due to resource shortage
+    3. System maintenance conducted improperly (for instance, multiple USSs bring down nodes contributing to the same range for maintenance simultaneously)
+12. Everything is working properly, but the system lacks the capacity to handle the volume of traffic
+
+### Check detection capabilities
+
+<table>
+    <tr>
+        <th rowspan="2">Check (+readiness)</th>
+        <th colspan="12">Failure</th>
+    </tr>
+    <tr>
+        <th>1</th>
+        <th>2</th>
+        <th>3</th>
+        <th>4</th>
+        <th>5</th>
+        <th>6</th>
+        <th>7</th>
+        <th>8</th>
+        <th>9</th>
+        <th>10</th>
+        <th>11</th>
+        <th>12</th>
+    </tr>
+    <tr>
+        <td>🚀 /healthy</td>
+        <td>✅</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+    </tr>
+    <tr>
+        <td>🛠️ Normal usage metrics</td>
+        <td>✅</td>
+        <td>✅</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>✅</td>
+        <td>✅</td>
+        <td>❌</td>
+        <td>🔶</td>
+        <td>❌</td>
+        <td>✅</td>
+        <td>🔶</td>
+        <td>🔶</td>
+    </tr>
+    <tr>
+        <td>✅ DAR identity</td>
+        <td>✅</td>
+        <td>✅</td>
+        <td>✅</td>
+        <td>✅</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>🔶</td>
+        <td>🔶↓↓</td>
```

Comment:
Is the double ↓ intended here?

Reply:
Yeah, that's something like "very rarely" -- I'll add it explicitly to the legend.